R/prep_priors.R

```diff
@@ -5,35 +5,49 @@
 #' 
 #' @param max_antigens [integer]: how many antigen-isotypes will be modeled
 #' @param priors a [list] with optional elements:
-#' - "mu_hyp_param": hyperpriors for estimate parameters. If specified must
+#' - "mu_hyp_param": A [matrix] of hyperpriors for parameter
+#' fixed effects.
+#' If specified must
 #'    be 5 values long.
 #'    - y0 = baseline antibody concentration (default = 1.0)
 #'    - y1 = peak antibody concentration (default = 7.0)
 #'    - t1 = time to peak (default = 1.0)
 #'    - r = shape parameter (default = -4.0)
 #'    - alpha = decay rate (default = -1.0)
-#' - "prec_hyp_param": Precision corresponding to mu_hyp_param. If specified
-#' must 5 values long.
+#' - "prec_hyp_param": Corresponds to diagonal entries representing the
+#' precision [matrix] of hyper-parameter for person-specific fixed effects.
```

Comment:
This instance of "matrix" is referring to the mathematical concept, not the R object class:

```suggestion
#' precision matrix of hyper-parameter for person-specific fixed effects.
```